midi_app_controller/gui/binds_editor.py

```diff
@@ -291,17 +397,26 @@ def __init__(
             controller.knobs,
             binds.knob_binds,
             actions,
+            connected_controller,
         )
         self.buttons_widget = ButtonBinds(
             controller.buttons,
             binds.button_binds,
             actions,
+            connected_controller,
         )
 
         # Layout.
         layout = QVBoxLayout()
         layout.addLayout(radio_layout)
         layout.addLayout(buttons_layout)
+
+        handling_controller = connected_controller is not None
+        if not handling_controller:
```

Comment:
It's a bit confusing when you use `not` on `not`. Maybe just `if connected_controller is None:`?

Reply:
I changed it, because i thought you wanted the code to be more readable. Then i realized you meant the prompt for the user, and forgot to change it back.